lib/util.py

```diff
@@ -3,7 +3,7 @@
 
 
 def has_common_element(list1, list2):
-    return any(e in list2 for e in list1)
+    return True if set(list1) & set(list2) else False
```

Comment:
This should also work

```python
return bool(set(list1) & set(list2))
```

which would be read as "return whether or not these two sets intersect"

or

```python
return len(set(list1) & set(list2)) > 0
```

But idc, up to you :D